requirements-dev.txt

```diff
@@ -0,0 +1,14 @@
+pytest
+pytest-cov
+mock
+pymysql
+pymssql
+psycopg2
+pymssql
+psycopg2
+flake8
+# FIXME: I am not sure these are needed.
```

Comment:
It looks like you have left in all of the dependencies that I usually install when running the API, so this looks good to me.